rules/scripts/hsmtool_runner.template.sh

```diff
@@ -168,6 +168,8 @@ certgen () {
       -CAkeyform engine \
       -CAkey "pkcs11:pin-value=${HSMTOOL_PIN};object=${endorsing_key};token=${FLAGS_HSMTOOL_TOKEN}"
   fi
+  echo "Converting certificate for ${ca_key} to DER"
```

Comment:
@moidx we need to be able to generate the ICA certs with the Luna HSM so that the integration tests can pass (and the SPM just parses the cert to get the serial number out). The generic pkcs11 openssl engine does not work with Luna it seems. I think we need to install the Gem engine on the CI machine: https://www.thalesdocs.com/gphsm/integrations/guides/openssl/index.html#integrate-openssl-with-luna-hsm-on-unix for this to work